scripts/build_pdo_snowflake.sh

```diff
@@ -65,3 +65,4 @@ cc -shared \
 source $DIR/env.sh
 
 env | grep SNOWFLAKE_TEST > $DIR/../testenv.ini
+echo "max_execution_time = 0" >> $DIR/../testenv.ini
```

Comment:
I think we can get rid of this line since we know that `max_execution_time` was not the culprit of the 7.0/7.1 failures